### ISSUE TITLE
Quicksand breaks the game

### ISSUE THREAD

**Issue by maddenbrain** [mtgred/netrunner#1683]
Please describe the circumstances of your error here.
Quicksand breaks the game.

Stack Trace:
```clojure
#error {
 :cause nil
 :via
 [{:type java.lang.NullPointerException
   :message nil
   :at nil}]
 :trace
 []}
```

**Comment by JoelCFC25** [comment]
To be clear, was this immediately when you try to rez it? Should've been fixed at #1671, but maybe that isn't live yet? Make sure you've done a forced refresh if it's been a few days since you visited the site.

**Comment by JoelCFC25** [comment]
The fix should be live now.
